Daytime-Operations/Auxiliary-Telescope/AuxTel-LOTO-procedure.rst

```diff
@@ -0,0 +1,129 @@
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Alysha Shugart*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Safety Team*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _Auxiliary-Telescope-AuxTel-LOTO-procedure:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+#####################
+AuxTel LOTO Procedure
+#####################
+
+.. _AuxTel-LOTO-procedure-Overview:
+
+Overview
+^^^^^^^^
+
+.. This section should provide a brief, top-level description of the procedure's purpose and utilization. Consider including the expected user and when the procedure will be performed.
+
+AuxTel must be under LOTO (lock-out tag-out) any time physical work is undertaken on the mount or the dome. 
+Working with environmental sensors, on the truss, mirrors, or anywhere in the dome where unexpected movement could cause a safety hazard, the telescope must be locked out. 
+Trained summit support staff may use this procedure if they meet all of the preconiditon requirements. 
+
+.. _AuxTel-LOTO-procedure-Precondition:
+
+Precondition
+^^^^^^^^^^^^
+
+.. This section should provide simple overview of preconditions before executing the procedure; for example, state of equipment, telescope or seeing conditions or notifications prior to execution.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. If there is a different procedure that is critical before execution, carefully consider if it should be linked within this section or as part of the Procedure section below (or both).
+
+Before putting the AuxTel under LOTO, you must have completed the :ref:`LOTO safety training <Daytime-Operations-Safety-Control-Safety-Systems-LOTO>` and familiarized yourself with the :ref:`Control Safety Systems <Daytime-Operations-Safety-Control-Safety-Systems>` document. 
+You should have your own unique lock and key with your information identified, and this work must be coordinated with the day crew. 
+
+- The telescope must be parked in a safe position, and all CSCs in STANDBY. 
+
+    - If the ATDome is needed while the AuxTel is under LOTO, this must be verified with the safety team and day crew.
```

Comment:
Does this mean that the ATDome can be moved while the AuxTel is under LOTO? I think so, so we might need to be more explicit about that early on in the procedure, maybe even in the overview section above.

Reply:
Clarified. 👍